<launch>
  <arg name="uav_id" default="1"/>
  <arg name="cmd_sub_topic" default="/position_cmd"/>
  <arg name="control_pub_topic" default="/uav1/sunray/uav_control_cmd"/>

  <node name="ego2sunray_node" pkg="sunray_simulator" type="ego2sunray" output="screen">
    <param name="uav_id" value="$(arg uav_id)"/>
    <param name="cmd_sub_topic" value="$(arg cmd_sub_topic)"/>
    <param name="control_pub_topic" value="$(arg control_pub_topic)"/>
  </node>
</launch>
